.github/workflows/sync-upstream-advanced.yml

```
@@ -0,0 +1,105 @@
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)